webapp/components/update_command_modal.jsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {Modal} from 'react-bootstrap';
+
+import {FormattedMessage} from 'react-intl';
+
+import React from 'react';
+
+export default class UpdateCommandModal extends React.Component {
```

Comment:
I actually think you can use our existing `confirm_modal.jsx` in place of this one, can you see if that's feasible to do?